mindjunkies/forums/views.py

```
@@ -7,9 +7,10 @@
 from mindjunkies.courses.models import Course, Module
 
 from .forms import ForumCommentForm, ForumReplyForm, ForumTopicForm
-from .models import ForumComment, ForumTopic, Reply
-
+from .models import Course, ForumComment, ForumTopic, Reply
 
+from django.http import HttpResponse
```

Ruff (0.8.2)

`django.http.HttpResponse` imported but unused

Remove unused import: `django.http.HttpResponse`

(F401)